satpy/tests/reader_tests/test_osisaf_l3.py

```diff
@@ -0,0 +1,364 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Those lines are redundant and can be removed.

Reply:
Fixed!